.github/actions/publish-nuget/action.yml

```
@@ -39,6 +42,12 @@ inputs:
     required: false
     default: 'true'
     description: If set to "true", the Microsoft.SourceLink.GitHub NuGet package is added to the projects.
+  dry-run:
+    required: false
+    default: 'false'
+    description: |
+      If set to "true", the action will perform all package creation and validation steps without actually publishing to
+      NuGet or creating a GitHub release.
```

It's always a NuGet package, but you don't necessarily publish _to_ NuGet.
```suggestion
      If set to "true", the action will perform all package creation and validation steps without actually publishing
      the NuGet package or creating a GitHub release.
```